src/components/chat.tsx

```diff
@@ -774,3 +857,17 @@ function FeedbackButtons(props: {
     </div>
   );
 }
+
+const RecordingDots = () => (
+  <div className="sf-flex sf-justify-center sf-items-center sf-space-x-1">
+    {[...Array(3)].map((_, i) => (
+      <span
+        key={i}
+        className="sf-bg-white sf-rounded-full sf-w-2 sf-h-2 sf-animate-bounce"
+        style={{
+          animationDelay: `${i * 0.2}s`,
```

Comment:
Tried to push an addition to this PR because the lovely delay wasn't working on my version. It didn't let me push to your fork, so here is the diff of changes I made:

```
@@ -863,10 +863,14 @@ const RecordingDots = () => (
     {[...Array(3)].map((_, i) => (
       <span
         key={i}
-        className="sf-bg-white sf-rounded-full sf-w-2 sf-h-2 sf-animate-bounce"
-        style={{
-          animationDelay: `${i * 0.2}s`,
-        }}
+        className={classNames(
+          "sf-bg-white sf-rounded-full sf-w-2 sf-h-2 sf-animate-bounce",
+          i === 0
+            ? ""
+            : i === 1
+            ? "sf-animation-delay-02"
+            : "sf-animation-delay-04",
+        )}
       ></span>
     ))}
   </div>
diff --git a/src/index.css b/src/index.css
index 848df7e..c57366e 100644
--- a/src/index.css
+++ b/src/index.css
@@ -11,6 +11,15 @@
   }
 }
 
+.sf-animation-delay-02 {
+    animation-delay: 0.2s !important;
+}
+
+.sf-animation-delay-04 {
+    animation-delay: 0.4s !important;
+}
+
```

Reply:
would you mind adding this? Slightly confused by the pasted text haha

Reply:
Yeah I can do it :smile: